packages/sdk-core/src/model/configuration/BacktraceDatabaseConfiguration.ts

```diff
@@ -2,7 +2,7 @@ export interface EnabledBacktraceDatabaseConfiguration {
     /**
```

Comment:
Should these methods be EnableBacktraceDatabaseConfiguration and DisableBacktraceDatabaseConfiguration without the d?

Reply:
I have mixed feelings about that. In the end the result should be the configuration for enabled database